apps/editor/src/app/atoms/modelTitle.tsx

```diff
@@ -86,11 +104,14 @@ export function ModelTitle({
       {/* title */}
       <FlexboxGrid.Item colspan={12}>
         <FlexboxGrid align="middle">
-          <Link to={closePath}>
-            <ChevronLeft />
-          </Link>
-
-          <h1>{titleView()}</h1>
+          <FlexRow>
+            <PaddedCol xs={29}>
+              <FlexLink to={closePath}>
+                <ChevronLeft />
+              </FlexLink>
+              <h2>{titleView()}</h2>
```

Comment:
```suggestion
              <h1>{titleView()}</h1>
```

Reply:
<h1> is huge. Doesn't look nice imo ;)

Reply:
Styling is independent of the element. The title is semantically a h1.